From: Jens Osterkamp <Jens.Osterkamp@de.ibm.com>
Subject: fix spidernet build issue
Patch-mainline: 2.6.17-rc1

WARNING: drivers/net/sungem_phy: duplicate symbol 'mii_phy_probe' previous definition was in drivers/net/spidernet.ko

Signed-off-by: Jens Osterkamp <Jens.Osterkamp@de.ibm.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
Signed-off-by: Olaf Hering <olh@suse.de>
---

 drivers/net/Makefile |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff -puN drivers/net/Makefile~fix-spidernet-build-issue drivers/net/Makefile
--- 25/drivers/net/Makefile~fix-spidernet-build-issue	Fri Mar 10 14:36:34 2006
+++ 25-akpm/drivers/net/Makefile	Fri Mar 10 14:36:34 2006
@@ -58,8 +58,8 @@ obj-$(CONFIG_STNIC) += stnic.o 8390.o
 obj-$(CONFIG_FEALNX) += fealnx.o
 obj-$(CONFIG_TIGON3) += tg3.o
 obj-$(CONFIG_BNX2) += bnx2.o
-spidernet-y += spider_net.o spider_net_ethtool.o sungem_phy.o
-obj-$(CONFIG_SPIDER_NET) += spidernet.o
+spidernet-y += spider_net.o spider_net_ethtool.o
+obj-$(CONFIG_SPIDER_NET) += spidernet.o sungem_phy.o
 obj-$(CONFIG_TC35815) += tc35815.o
 obj-$(CONFIG_SKGE) += skge.o
 obj-$(CONFIG_SKY2) += sky2.o
_
